design/account.go

```diff
@@ -170,7 +170,7 @@ var userDataAttributes = a.Type("UserDataAttributes", func() {
 	a.Attribute("url", d.String, "The url")
 	a.Attribute("company", d.String, "The company")
 	a.Attribute("providerType", d.String, "The IDP provided this identity")
-	a.Attribute("cluster", d.String, "The cluster where the user has her OpenShift account is created")
+	a.Attribute("cluster", d.String, "The OpenShift API URL of the cluster where the user's tenant services are being run")
```

Comment:
Or rather "The OpenShift API URL of the cluster where the user was provisioned to". The tenant service is an implementation detail. The clients (the registration app) doesn't have to know such details.